crates/esplora/tests/blocking_ext.rs

```diff
@@ -239,14 +239,14 @@ pub fn test_update_tx_graph_gap_limit() -> anyhow::Result<()> {
         sleep(Duration::from_millis(10))
     }
 
-    // A scan with gap limit 4 won't find the second transaction, but a scan with gap limit 5 will.
+    // A scan with gap limit 4 won't find the second transaction, but a scan with gap limit 6 will.
     // The last active indice won't be updated in the first case but will in the second one.
     let (graph_update, active_indices) = env.client.full_scan(keychains.clone(), 4, 1)?;
```

Comment:
```suggestion
    // A scan with gap limit 5 won't find the second transaction, but a scan with gap limit 6 will.
    // The last active indice won't be updated in the first case but will in the second one.
    let (graph_update, active_indices) = env.client.full_scan(keychains.clone(), 5, 1)?;
```

nit:

If the new code is right, we will want to now see that a scan with gap limit 5 won't find the second transaction.

Reply:
Done in b9ba9b5978c2dfb1ee4cd29ed565365bd62ee75f

Reply:
This wasn't resolved, see my comment